generator/srtm_coverage_checker/srtm_coverage_checker.cpp

```diff
@@ -2,56 +2,38 @@
 
 #include "map/feature_vec_model.hpp"
 
-#include "routing/routing_integration_tests/routing_test_tools.hpp"
+#include "routing/routing_helpers.hpp"
 
+#include "indexer/classificator.hpp"
+#include "indexer/classificator_loader.hpp"
+#include "indexer/feature.hpp"
 #include "indexer/feature_altitude.hpp"
+#include "indexer/feature_data.hpp"
+#include "indexer/feature_processor.hpp"
 
 #include "coding/file_name_utils.hpp"
 
 #include "platform/country_file.hpp"
+#include "platform/local_country_file.hpp"
 #include "platform/local_country_file_utils.hpp"
+#include "platform/platform.hpp"
 
 #include "base/logging.hpp"
 
 #include <algorithm>
-
-#include <sys/resource.h>
+#include <cstdint>
+#include <limits>
+#include <string>
 
 #include "3party/gflags/src/gflags/gflags.h"
 
 DEFINE_string(srtm_path, "", "Path to directory with SRTM files");
 DEFINE_string(mwm_path, "", "Path to mwm files (writable dir)");
 
-// TODO: remove when routing_test_tools becomes a library.
-namespace
-{
-void ChangeMaxNumberOfOpenFiles(size_t n)
-{
-  struct rlimit rlp;
-  getrlimit(RLIMIT_NOFILE, &rlp);
-  rlp.rlim_cur = n;
-  setrlimit(RLIMIT_NOFILE, &rlp);
-}
-
-shared_ptr<model::FeaturesFetcher> CreateFeaturesFetcher(vector<LocalCountryFile> const & localFiles)
-{
-  size_t const maxOpenFileNumber = 1024;
-  ChangeMaxNumberOfOpenFiles(maxOpenFileNumber);
-  shared_ptr<model::FeaturesFetcher> featuresFetcher(new model::FeaturesFetcher);
-  featuresFetcher->InitClassificator();
-
-  for (LocalCountryFile const & localFile : localFiles)
-  {
-    auto p = featuresFetcher->RegisterMap(localFile);
-    if (p.second != MwmSet::RegResult::Success)
-    {
-      ASSERT(false, ("Can't register", localFile));
-      return nullptr;
-    }
-  }
-  return featuresFetcher;
-}
-}  // namespace
+using namespace feature;
+using namespace platform;
+using namespace routing;
+using namespace std;
```

Comment:
Where do you use std namespace here?

Reply:
vector/numeric_limits. Do you propose to use std:: instead of using namespace std;?

Removed: `#include <algorithm>` and `#include <string>` as well.